sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/HomeScreen.kt

```diff
@@ -71,9 +70,9 @@ private fun HomeScreenContent(
     }
 }
 
-@Preview(showBackground = true)
 @Composable
-private fun HomeComposeScreenPreview() {
+@Preview(showSystemUi = true)
+private fun HomeScreenPreview() {
```

Comment:
@luongvo Please follow our convention for the order of annotation.

https://nimblehq.co/compass/development/android/user-interface/jetpack-compose/#ordering-of-annotations-and-preview-functions

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/372/commits/27d967ee14c09dccbc4787ea2269d31cfad43d24